app/views/api/v1/crops/index.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.data @crops do |crop|
+  json.(crop, :id, :name)
+  json.set! '@id', url_for(:only_path => false) + '/' + crop.id.to_s
```

Comment:
this is json-ld, right?

in re: our discussion about pretty URLs, it occurs to me that /type/id (eg. /crops/42 or /plantings/42) will be unchanging forever while the friendly URLs (as generated by the friendly_id gem) will tend to change anyway with name changes eg. /posts/unspecific-title to /posts/help-me-with-my-aphid-problem or whatever.  This is similar to wordpress keeping the ?p=nnn the same while changing the URL when you change a page title.

Anyway there's maybe a discussion to be had about having a "machine-friendly" URL that is unchangeable and using that in the API.